### ISSUE TITLE
Selected option attribute

### ISSUE THREAD

**Issue by tgoorden** [mquandalle/meteor-jade#149]
I'm wrestling with making a pre-selected option render properly. It seems like the "selected" attribute is somehow filtered out of option tags, even though it is a valid HTML4/5 attribute. Any ideas?

For example:
```
   each contexts
         if isContextSelected
            option(value="#{this}" selected="selected") #{this} (Selected)
         else
            option(value="#{this}") #{this}
```
The `"(Selected)"` label works fine, proving that the `isContextSelected` helper function is correct. However, the `selected` attribute does not show up in the DOM. I've tried all sorts of variations ($dyn attributes, using `selected="true"` or just `selected`), but it never shows up.

**Comment by stivaugoin** [comment]
Same problem here.

**Comment by jaschaephraim** [comment]
Just to add some information:

    option(value='1' selected='selected') Whatever

... works fine, but:

    if something_true
        option(value='1' selected='selected') Whatever

... will strip the selected attribute.

**Comment by dalgard** [comment]
You can simply write:

```jade
each contexts
  option(value=this selected=isContextSelected) #{this} (Selected)
```

**Comment by jaschaephraim** [comment]
The `selected` attribute is stripped in that case as well.

**Comment by dalgard** [comment]
If `isContextSelected` is `true`, it *should* result in a boolean `selected` attribute.

**Comment by jaschaephraim** [comment]
Yes, that's why it's an issue that it doesn't work (at least when I checked a few days ago). Whenever the `selected` attribute in particular was present with a variable or conditional or anything dynamic, it was stripped.

**Comment by dalgard** [comment]
Does it work with Blaze syntax (embedded in Jade)?

**Comment by jaschaephraim** [comment]
No, I assure you this is a valid issue.

```jade
option(selected='selected' someattr='selected')
option(selected=trueVar someattr=trueVar)
if trueVar
  option(selected='selected' someattr='selected')

| <option selected='selected' someattr='selected'></option>
| <option selected={{trueVar}} someattr={{trueVar}}></option>
| <option selected={{#if trueVar}}selected{{/if}} someattr={{#if trueVar}}selected{{/if}}></option>
```

results in:

```html
<option selected='selected' someattr='selected'></option>
<option someattr='true'></option>
<option someattr='selected'></option>
<option selected='selected' someattr='selected'></option>
<option someattr='true'></option>
<option someattr='selected'></option>
```

**Comment by dominickiller** [comment]
Is it possible to use two different functionality of option tag .... option "value" as well as "_id".... actually i have dynamically data in option tag so i want its "value" in some location and "_id" in some location??
